### ISSUE TITLE
false positive modulo operator

### ISSUE THREAD

**Issue by iham** [gforcada/flake8-pep3101#21]
```python
if var % 50 == 0:
    do something
```
leads to "S001 found module formatter" which would be used in
```python
var = u'a %s' % u'string'
```
or
```python
var = u'a %s and %s' % (u'string', u'another string')
```
or via a variable
```python
msg = u'string'
var = u'a %s' % msg
```
or multiple variables
```python
msg1 = u'string'
msg2 = u'another string'
var = u'a %s and %s' % (msg1, msg2)
```

i guess its work fiddling around with the regex to parse the modulo operator correct.
sadly i am not good at that ...

**Comment by gforcada** [comment]
@iham I fixed and made a new release: 1.2 would you mind giving it a try?

**Comment by iham** [comment]
i did & it works like a charm!
thank you very much!

your fast reaction is very appreciated!
